proj-android/PowerAuthLibrary/build.gradle

```diff
@@ -75,5 +79,21 @@ publishing {
     }
 }
 
-
-
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key  = System.getenv('BINTRAY_API_KEY')
+    publications = ['maven']
+	pkg {
+		repo = 'PowerAuth'
+		name = 'powerauth-android-sdk'
+		userOrg = 'lime-company'
```

Comment:
I think that new item like "ORG" from 'gradle.properties' used here will be nice ;)